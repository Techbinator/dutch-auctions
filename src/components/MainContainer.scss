.auth-container {
  max-width: 600px;
  margin: 10% auto;
  background-color: #fff;
  padding: 1%;
}

.container {
  width: 94%;
  margin: 0 auto;
  box-sizing: border-box;
  padding: 1%;
}
