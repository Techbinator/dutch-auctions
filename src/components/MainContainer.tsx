import React, { ReactNode } from "react";
import "./MainContainer.scss";
import Header from "../components/Header";
interface IMainContainer {
  isLogedIn: boolean;
  children: ReactNode;
}

export default ({ isLogedIn, children }: IMainContainer) => {
  return (
    <>
      <Header isLogedIn={isLogedIn} />
      <div
        className={`box-shadow ${isLogedIn ? "container" : "auth-container"}`}
      >
        {children}
      </div>
    </>
  );
};
