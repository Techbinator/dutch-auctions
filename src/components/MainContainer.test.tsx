import React from "react";
import MainContainer from "./MainContainer";
import { shallow } from "enzyme";

describe("MainContainer component", () => {
  it("should render snapshot with default values", () => {
    const wrapper = shallow(
      <MainContainer isLogedIn={true}>
        <div>children</div>
      </MainContainer>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
