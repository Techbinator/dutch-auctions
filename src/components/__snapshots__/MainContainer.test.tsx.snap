// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer component should render snapshot with default values 1`] = `
<Fragment>
  <withRouter(Header)
    isLogedIn={true}
  />
  <div
    className="box-shadow container"
  >
    <div>
      children
    </div>
  </div>
</Fragment>
`;
